### DISCUSSION TITLE
A question about the backup documentation

### DISCUSSION THREAD

**Discussion by FreeWind6** [immich-app/immich#14716] [General]
Good afternoon. I've read your backup documentation, but I still have questions.
1) In Postgresql, do you only use a database name immich?
2) Is it enough to make only backups of folders: library, upload, profile?
And the folders: thumbs and video - can I not make backups of them?
How will Immich react then if I try to restore it without these folders, because the data from these folders is in the database?
Will I need to create a task for transcoding videos and photos and then the old data from their database will be replaced with new ones?

I'm sorry if I wrote badly (I'm writing through a translator)

**Comment by alextran1502** [comment]
1. Yes
2. 
> is it enough to make only backups of folders: library, upload, profile?

Yes

> And the folders: thumbs and video - can I not make backups of them?

Yes

> How will Immich react then if I try to restore it without these folders, because the data from these folders is in the database?

You can run the jobs to recreate them

> Will I need to create a task for transcoding videos and photos and then the old data from their database will be replaced with new ones?

Yes

**Reply by FreeWind6** [reply]
Thanks a lot =)
